### PR TITLE
media-gfx/hydrus: Drop 534 and 538, Add 565

### PR DESCRIPTION
**Pull request by vaartis**
A dependency that would require a lot of new packages to be added ([psd-tools](https://github.com/psd-tools/psd-tools)) got introduced at some point that broke the tests for a while, so I kind of procrastinated on it until it was actually made optional.

The test fails with a wrong video length error, I'm not really sure how it's different, might be some kind of platform difference. I assume it's going to get fixed later, for now I made a commit on github and linked that into the ebuild.

Also reorganized the dependencies a bit.

### CODE REVIEW

**Review by juippis — COMMENTED** [review]
Sorry for the delay. Could you bump to latest version and I'll take a look?

**Comment by vaartis** [comment]
_Unfortunately_ either I was wrong about my assumption of it being actually made optional, or it was changed again. I'll net to patch some things to make it work again, because these dependencies would pull in about 5 or more packages that have very limited use.

**Comment by vaartis** [comment]
Terribly sorry this took so long, I kept putting it off. Patched the tests so they'd skip the annoying package, otherwise everything seems to be OK.

File: media-gfx/hydrus/hydrus-565.ebuild [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/hydrusnetwork/hydrus.git"
+else
+	SRC_URI="
+		https://github.com/hydrusnetwork/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+		https://github.com/vaartis/hydrus/commit/5d2505a1eb4721f7750861f7622c46adec31db8b.diff -> hydrus-3500-mpeg-error.patch
+		https://github.com/vaartis/hydrus/commit/dc20196f47ac29568a40cd04a45cf4fc5f237603.diff -> hydrus-test-fixes.patch
```
Comment (juippis) [inline media-gfx/hydrus/hydrus-565.ebuild:25]: I would prefer versioned distfiles, but hopefully these are merged to next release.

**Review by juippis — APPROVED** [review]
Better to have it in a good condition, thanks!

File: media-gfx/hydrus/hydrus-565.ebuild (hunk 1 above)
Comment (vaartis) [reply media-gfx/hydrus/hydrus-565.ebuild:25]: Unfortunately I'm not sure if these can be merged upstream, I have no idea what the first test is even supposed to do honestly, and the second one is for avoiding tests that require a pypi dependency that is just stupidly complicated to package.

**Pull request closed without merging**